tests/test_bgp_speaker.py

```diff
@@ -100,6 +100,13 @@ def common_setup_teardown(duthost, ptfhost):
                        peer_asn=mg_facts['minigraph_bgp_asn'],
                        port=str(port_num[i]))
 
+    for i in range(0, 3):
+        for count in range(0, 5):
+            time.sleep(15)
+            ret = ptfhost.exabgp(name="bgps%d" % i, state="status")
+            if 'RUNNING' is ret['status']:
+                break 
+
```

Comment:
What if the exabgp didn't start? Should we stopped the test?

Reply:
The test would fail. it would be better to leave the test and debug the failure rather than silently skipping it.

Reply:
I think it is better to add the checking logic in the exabgp module instead of letting application to handle it.

Reply:
Relocated in a separate PR